dbt_project.yml

```
@@ -19,3 +19,4 @@ models:
       materialized: table
       staging:
         materialized: view
+      tags: ["jaffle_shop", "dbt"]
```

```suggestion
        tags: ["staging", "hourly"]
```

I would:
1. just apply the tags to a subset of models (to test cases both with and without tags in the docs PR builder)
2. make these tags semantically meaningful, as we use this project as a reference for folks that are newer to dbt

---

Thank you.